pkg/gcv/cf/cf.go

```diff
@@ -231,3 +231,18 @@ func (cf *ConstraintFramework) Audit(ctx context.Context) (*validator.AuditRespo
 
 	return response, nil
 }
+
+// GetConstraint returns a constraint by name
+func (cf *ConstraintFramework) GetConstraint(ctx context.Context, name string) (*validator.Constraint, error) {
```

Comment:
can we add unit test for this? (also include a case for same name appearing under 2 different kinds)